shared/src/packages/trucks/libs/enums/truck-status.enum.ts

```diff
@@ -0,0 +1,8 @@
+const TruckStatus = {
+  NOT_AVAILABLE: 'Not available',
```

Comment:
it's only for UI, or will be used on backend?

Reply:
Particular `Not available` is not used on backend, so it will be deleted soon